src/Homework 4/towns.py

```diff
@@ -0,0 +1,53 @@
+"""Дан список стран и городов каждой страны. Затем даны названия
+городов. Для каждого города укажите, в какой стране он находится.
+Входные данные
+Программа получает на вход количество стран N. Далее идет N строк,
+каждая строка начинается с названия страны, затем идут названия
+городов этой страны. В следующей строке записано число M,
+далее идут M запросов — названия каких-то M городов, перечисленных выше.
+Выходные данные
+Для каждого из запроса выведите название страны, в котором находится
+данный город.
+Примеры
+Входные данные
+2
+Russia Moscow Petersburg Novgorod Kaluga
+Ukraine Kiev Donetsk Odessa
+3
+Odessa
+Moscow
+Novgorod
+
+Выходные данные
+Ukraine
+Russia
+Russia
+
+"""
+
+
+# Разбираем строку на города:страну, добавляем в словарь
+def get_towns(countries_q, ):
+    while countries_q:
+        input_towns = input('Введи строку: ').split(' ')
+        for town in input_towns[1:]:
+            if towns.setdefault(town, False):
```

Comment:
we have get for this. false is inconsistent for our  dict and no reasons to set it to False